client/src/pages/ExistingProgramme/index.js

```diff
@@ -30,15 +32,22 @@ const ExistingProgramme = ({ id }) => {
                 </Text>
             </Flex>
 
-            <Flex align="center" justify="center" marginTop="50px">
+            <Flex align="center" justify="center" direction="column">
                 <Text textAlign="center" fontSize="md" color="#000000" as="i">
                     Requirements
                 </Text>
-                <Box as="span" ml="2" color="gray.600" bg="gray" />
+                <Box bg="#e2e2e2" width="75%" textAlign="center">
```

Comment:
Can we please add some padding/margin so it's not so close to the title?

Reply:
done